cmd/controller-manager/app/controller-manager.go

```diff
@@ -245,6 +252,9 @@ func setInt(target *int, defaultValue int) {
 
 func setDefaultFederationConfig(fedConfig *corev1a1.FederationConfig) {
 	spec := &fedConfig.Spec
+	if spec.Scope == "" {
```

Comment:
(No action required) It was beaten into me early in my time working on openshift by no less than smarterclayton to prefer a len check vs an equality check to determine if a string is empty (e.g. `len(spec.Scope) == 0`).  I don't think we've been particularly rigorous in this codebase, though, so I can't enforce it.